sim/firesim-lib/src/main/scala/bridges/DromajoBridge.scala

```diff
@@ -46,7 +46,7 @@ class DromajoBridgeModule(key: TraceBundleWidths)(implicit p: Parameters) extend
     with StreamToHostCPU
 {
   // CONSTANTS: DMA Parameters
-  val toHostCPUQueueDepth = TokenQueueConsts.TOKEN_QUEUE_DEPTH
+  val toHostCPUQueueDepth = 6144
```

Comment:
Why is this decoupled from the `TokenQueueConsts.TOKEN_QUEUE_DEPTH`? (ditto for the TracerVBridge)